resources/views/categories/index.blade.php

```diff
@@ -0,0 +1,59 @@
+@extends('layouts.master')
+
+@section('main-content')
+    <div class="box">
+        <div class="box-header">
+            <h3 class="box-title">Data Table With Full Features</h3>
+            <button class="btn btn-primary" style="float: right" title="Add Category">
+                <span class="glyphicon glyphicon-plus"></span>
+            </button>
+        </div>
+        <!-- /.box-header -->
+        <div class="box-body">
+            <div id="list_category_wrapper" class="dataTables_wrapper form-inline dt-bootstrap">
+                <div class="row">
+                    <div class="col-sm-12">
+                        <table id="list_category" class="table table-bordered table-striped dataTable table-hover" role="grid"
```

Comment:
`id="list_category"` use dash style for naming html id and class, `id="list-category"`
Please fix for all